eng/build.sh

```diff
@@ -524,6 +524,16 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
 
+      -skipcrossarchnative)
+      arguments="$arguments /p:SkipCrossArchNative=true"
+      shift 1
+      ;;
+
+      -verbosity)
+      arguments="$arguments /p:Verbose=true"
```

Comment:
hm, the parameter should be verbose, to match the native script. Also, the transport property name (Verbose) is a very common name - maybe call it CoreclrVerbose until others start using it?

Reply:
Hi @hoyosjs,
I've just updated this commit, removed the other commit about -skipcrossarchnative options and added a commit to drop comment about -skipcrossarchnative.